src/components/navbar/NavBar.jsx

```diff
@@ -1,38 +1,82 @@
 import React from 'react';
```

Comment:
Can you copy the entire contents of this file and past it into index.js and then delete this file, decided to go with a different file structure